<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>247</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>55</number>
   </property>
   <property name="rightMargin">
    <number>55</number>
   </property>
   <property name="horizontalSpacing">
    <number>11</number>
   </property>
   <property name="verticalSpacing">
    <number>7</number>
   </property>
   <item row="6" column="0">
    <widget class="QPushButton" name="pb_cancel">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="spB_port">
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Myriad Pro</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Имя хоста</string>
     </property>
     <property name="buddy">
      <cstring>le_host</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Myriad Pro</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Протокол подключения</string>
     </property>
     <property name="buddy">
      <cstring>spB_port</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="le_login"/>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pb_ok">
     <property name="text">
      <string>Ок</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_dbName"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="le_host"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="le_pass">
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Myriad Pro</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Имя БД</string>
     </property>
     <property name="buddy">
      <cstring>le_dbName</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Myriad Pro</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Логин</string>
     </property>
     <property name="buddy">
      <cstring>le_login</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Myriad Pro</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Пароль</string>
     </property>
     <property name="buddy">
      <cstring>le_pass</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_host</tabstop>
  <tabstop>le_dbName</tabstop>
  <tabstop>le_login</tabstop>
  <tabstop>le_pass</tabstop>
  <tabstop>spB_port</tabstop>
  <tabstop>pb_cancel</tabstop>
  <tabstop>pb_ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
